mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -14,34 +15,100 @@
  */
 --%>
 
-<%@ include file="/init.jsp" %>
+<%@ include file="init.jsp" %>
 
 <div class="list-container">
 	<div class="header">
 		<h2>
 			<liferay-ui:message key="to-do-list" />
-
 			<i class="icon-check"></i>
 		</h2>
 	</div>
+	<div class="welcome-message">
+		<liferay-ui:message key="welcome" />
```

Comment:
This indentation and spacing convention is in general:
1. Sibling nodes are separated by a blank row, with the same indentation.
1. Child nodes are indented by one tab and are not separated by a blank row.

There are 100+ places in this PR that this needs to be applied.